xml/selinux.xml

```diff
@@ -402,11 +400,11 @@ system_u:object_r:var_t var</screen>
    </para>
    <para>
     Before switching into enforcing mode, make sure to first reset the
-    security context (extended attributes). Do so by executing
-    <command>restorecon -R /*</command> and then reboot the system.
+    security context (extended attributes):
    </para>
-   </warning>
-   <para>
+  <screen>&prompt.sudo;<command>restorecon -R /*</command></screen>
+
+  <!--<para>
     After logging in, run the <command>sestatus -v</command> command.
```

Comment:
@jsegitz : Where to move this example now (I commented it for now because with the latest change it does not make sense here any more)? Should I move it after l. 450-451? If yes, would it be enough to change the values of ''Current mode' and 'Mode from config file' in the example from 'permissive' to 'enforcing' then? (because the user would run this command now when already being in enforcing mode).

Reply:
@jsegitz : Latest PDF build: 
[cha-selinux_draft_20230103_2.pdf](https://github.com/SUSE/doc-sle/files/10338206/cha-selinux_draft_20230103_2.pdf)